tests/db/test_db_schema.py

```diff
@@ -261,6 +261,8 @@ def make_versions(idx, *fcts):
 async def test_dbschema_get_dct_with_update_functions():
     module_names = [modname for _, modname, ispkg in pkgutil.iter_modules(data.PACKAGE_WITH_UPDATE_FILES.__path__) if not ispkg]
     all_versions = [int(mod_name[1:]) for mod_name in module_names]
+    # The latest one should be disabled
+    all_versions.pop()
```

Comment:
The latest version will not always be disabled. It will only be disabled when no schema updates are included in the upcoming release up to now.